### PR TITLE
Cleanup DB code

### PR DESCRIPTION
**Pull request by markdboyd**

### CODE REVIEW

File: packages/db/src/database.ts [hunk 1]
```diff
@@ -6,12 +6,6 @@ import { tableNames } from './tables';
 
 export const isRecordDefined = <T>(record: T) => record !== undefined;
 
-export const doesRecordExist = async<T>(
-  params: Partial<T>,
-  knex: Knex,
-  tableName: string
-): Promise<boolean> => isRecordDefined(await knex<T>(tableName).where(params).first());
-
```
Comment (botanical) [inline packages/db/src/database.ts:14]: Should you remove `getRecordCumulusId` as well?

Comment (markdboyd) [reply packages/db/src/database.ts:14]: Yup, good catch!

Comment (markdboyd) [reply packages/db/src/database.ts:14]: 46572d3010a007041ed4b38d9098800f68513566

File: packages/api/src/lib/granule-delete.d.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+export {};
+//# sourceMappingURL=granule-delete.d.ts.map
```
Comment (botanical) [inline packages/api/src/lib/granule-delete.d.ts:2]: What is this file?

Comment (markdboyd) [reply packages/api/src/lib/granule-delete.d.ts:2]: A compiled TS file that I definitely didn't mean to commit. I'll fix

Comment (markdboyd) [reply packages/api/src/lib/granule-delete.d.ts]: 3136e699e4bd7953ba57aaaab21286b4383ca2f5

File: packages/db/src/database.ts [hunk 3]
```diff
@@ -5,33 +5,3 @@ import { RecordDoesNotExist } from '@cumulus/errors';
 import { tableNames } from './tables';
```
Comment (botanical) [inline packages/db/src/database.ts:5]: I think you can remove these imports now

**Review by botanical — APPROVED** [review]
I have one comment about removing the imports in `packages/db/src/database.ts`, but other than that ✅

**Pull request merged**